packages/prisma/schema.prisma

```diff
@@ -225,6 +227,8 @@ model User {
   routingForms         App_RoutingForms_Form[] @relation("routing-form")
   verifiedNumbers      VerifiedNumber[]
   hosts                Host[]
+  /// @zod.custom(imports.bookerLayouts)
+  defaultBookerLayouts Json?
```

Comment:
Could we add this to users metadata?

Reply:
Same question as for the events. Let's wait for Omar's / Alex' response, then I'll apply it do both.

Reply:
Replied